SaintCoinach/Definitions/Action.json

```diff
@@ -110,39 +110,39 @@
     },
     {
       "index": 31,
-      "name": "Cost{Type}[1]"
+      "name": "PrimaryCost{Type}"
```

Comment:
These will be breaking in SC object model and xivapi, should probably be merged with 5.0 changes.

Reply:
fixed in SC, wasn't aware it was duplicated there. in any case, should be merged before 5.0 hits so it can be consolidated with other changes at the same time otherwise it's a nightmare waiting to happen